public/wp-content/themes/glific/page-get-started.php

```diff
@@ -36,14 +36,31 @@
                         <?php endforeach;
                         endif;
                         if ($key === 2) : ?>
-                        <a href="<?php $section_one_data['contact_us_url']; ?>" class="d-block w-154 text-decoration-none bg-theme-primary text-white font-heebo-regular fz-18 px-10 py-4 leading-27 rounded-top-15 rounded-bottom-left-15">Contact Us</a>
+                            <a href="<?php $section_one_data['contact_us_url']; ?>" class="d-block w-154 text-decoration-none bg-theme-primary text-white font-heebo-regular fz-18 px-10 py-4 leading-27 rounded-top-15 rounded-bottom-left-15">Contact Us</a>
                         <?php endif; ?>
                         </div>
                     </div>
                 </div>
             <?php endforeach;
             endif; ?>
     </div>
+    <?php $section_two_data = get_field('section_two');
```

Comment:
Same thoughts here, need to revisit for better naming which is more relevant to the section. Please update.